ScintSuite/SimulationCodes/SINPA/__init__.py

```diff
@@ -13,3 +13,4 @@
 from ScintSuite.SimulationCodes.SINPA._Orbits import OrbitClass as orbits
 import ScintSuite.SimulationCodes.SINPA._geometry as geometry
 from ScintSuite.SimulationCodes.SINPA._INPA_strike_points import INPAStrikes
+import ScintSuite.SimulationCodes.SINPA._Forw_Mod_xarray as fmxarray
```

Comment:
just call it forwardModelling, fm... o whatever you like most. Remove the xarray as this will become the official forward modelling module in the near future once it is merged to master